src/inference/src/dev/threading/istreams_executor.cpp

```diff
@@ -499,5 +501,42 @@ IStreamsExecutor::Config IStreamsExecutor::Config::make_default_multi_threaded(c
     return streamExecutorConfig;
 }
 
+IStreamsExecutor::Config IStreamsExecutor::Config::reserve_cpu_threads(const IStreamsExecutor::Config& initial) {
+    auto config = initial;
+    std::vector<int> stream_ids;
+    std::string log = "[ threading ] reserve_cpu_threads " + config._name;
```

Comment:
Could we disable log by default and add log only when LOG_LEVEL is changed?

Reply:
Yes, replaced this with OPENVINO_DEBUG. Removed this statis log, because it is duplicate with streams_info_table log.